google/datalab/bigquery/_table.py

```diff
@@ -636,46 +636,6 @@ def to_dataframe(self, start_row=0, max_rows=None):
     ordered_fields = [field.name for field in self.schema]
     return df[ordered_fields] if df is not None else pandas.DataFrame()
 
-  def to_file(self, destination, format='csv', csv_delimiter=',', csv_header=True):
```

Comment:
Made it private and updated _query.py.  Why did we decide to remove to_file() but not to_dataframe()?